partner_brand/views/report_template.xml

```diff
@@ -0,0 +1,80 @@
+<odoo>
+
+    <!-- Copyright (C) 2019 Open Source Integrators
+         License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+    <template id="external_layout_standard_logo"
```

Comment:
I thought having a layout per brand would be easier to handle in use cases where the header and footer of the report would be completely different in look and content.

override [`web.external_layout`](https://github.com/odoo/odoo/blob/12.0/addons/web/views/report_templates.xml#L342]) to have something like this
```xml
<t t-if="'brand_id' in o.fields_get() and o.brand_id and o.brand_id.external_layout_id">
	<t t-call="{{o.brand_id.external_layout_id.key}}"/>
</t>
<t t-else="">
    <t 	t-if="company.external_report_layout_id" t-call="{{company.external_report_layout_id.key}}">
    	<t t-raw="0"/>		
    </t>
    <t t-else="else" t-call="web.external_layout_standard">
    	<t t-raw="0"/>
    </t>
</t>
```

Reply:
@sbejaoui It would be nice to do that but Odoo has the header and footer and their elements in the same template. In your example, we would have to copy the whole template and adapt it in `o.brand_id.external_layout_id.key`. We chose a more surgical approach where we only inherits and modifies the elements we need. I guess it will make the maintenance and migration easier, we'll see.

Reply:
I think we can merge the two ideas:

- Override the `external_layout_standard` to handle brand details in the standard layout.
- Override the external_layout and call `o.brand_id.external_layout_id` if it is set in case we need to create a whole different design.

Reply:
#758